cmds/modules/noded/main.go

```diff
@@ -193,6 +195,27 @@ func action(cli *cli.Context) error {
 		return hypervisor, nil
 	})
 
+	log.Info().Msg("start perf scheduler")
+	perfMon, err := perf.NewPerformanceMonitor("unix://var/run/redis.sock")
+	if err != nil {
+		return errors.Wrap(err, "failed to create a new perfMon")
+	}
+	defer perfMon.Close()
+
+	err = perfMon.Run(ctx)
+	if err != nil {
+		log.Error().Err(err).Msg("failed to run the scheduler")
+	}
```

Comment:
```suggestion
	
	if err := perfMon.Run(ctx); err != nil {
		log.Error().Err(err).Msg("failed to run the scheduler")
	}
```

Also i think it's better if we return the error not just log